src/app/components/Search/ResultsList.jsx

```diff
@@ -17,12 +17,12 @@ const ResultsList = (results) => {
           results.results.map((result, i) => (
             <li className="nypl-results-item" key={i.toString()}>
               <h3>
-                <a href={`/work/detail/${result._id}`}>
+                <a href={`/work/${result._id}`}>
```

Comment:
This is good but you could also try to use react-router's `<Link>` component to link to different states in the app.

Reply:
Not needed for this PR.

Reply:
reverting this change and will make adjustments in the next PR which will be work on the item pages.